VideoLocker/res/layout/row_course_dashboard_list.xml

```diff
@@ -1,36 +1,36 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    android:id="@+id/chapter_row_layout"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/chapter_row_layout"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/dashboard_navigation_element_height"
+    android:layout_height="@dimen/dashboard_nav_element_height"
```

Comment:
after looking at code, this change won't be needed cuz both the `TextViews` are already being wrapped to only occupy 1 line.

TextView # 1: https://github.com/edx/edx-app-android/pull/434/files#diff-c79807dabf60d94b7a3a6e12ab19a449R42

TextView # 2: https://github.com/edx/edx-app-android/pull/434/files#diff-c79807dabf60d94b7a3a6e12ab19a449R54